clients/py/tests/conftest.py

```diff
@@ -43,6 +43,7 @@
 
 extra_backends = [
     {'type': 'kv'},
+    {'type': 'mysql'},
```

Comment:
did you mean nosql?

f.        [#5] Add a "nosql" alias to the "kv" backend type to match the terminology used in the docs. (I suggest adding an alias rather than renaming to avoid breaking backwards compatibility.)

Reply:
oh no lol
i'll fix it